lib/ui/views/dashboard_view.dart

```diff
@@ -0,0 +1,205 @@
+                dismissCard(model, element);
+              },
+              child: Text(element.toString(), key: UniqueKey())));
+          break;
+        case PreferencesFlag.progressBarCard:
+          cards.add(Dismissible(
+              key: UniqueKey(),
+              onDismissed: (DismissDirection direction) {
+                dismissCard(model, element);
+              },
+              child: Text(element.toString())));
+          break;
+        case PreferencesFlag.gradesCards:
+          cards.add(_buildGradesCards(model, element));
+          break;
+        default:
+      }
+    }
+
+    return cards;
+  }
+
+  Widget _buildAboutUsCard(DashboardViewModel model, PreferencesFlag flag) =>
+      DismissibleCard(
+        key: UniqueKey(),
+        onDismissed: (DismissDirection direction) {
+          dismissCard(model, flag);
+        },
+        cardColor: AppTheme.appletsPurple,
+        child: Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(AppIntl.of(context).card_applets_title,
+                  style: Theme.of(context).primaryTextTheme.headline6),
+              const SizedBox(height: 10),
+              Text(AppIntl.of(context).card_applets_text,
+                  style: Theme.of(context).primaryTextTheme.bodyText2),
+              const SizedBox(height: 10),
+              Wrap(spacing: 15.0, children: [
+                TextButton(
+                  onPressed: () {
+                    Utils.launchURL(Urls.clubFacebook, AppIntl.of(context));
+                  },
+                  child: Text(AppIntl.of(context).facebook.toUpperCase(),
+                      style: Theme.of(context).primaryTextTheme.button),
+                ),
+                TextButton(
+                  onPressed: () {
+                    Utils.launchURL(Urls.clubGithub, AppIntl.of(context));
+                  },
+                  child: Text(AppIntl.of(context).github.toUpperCase(),
+                      style: Theme.of(context).primaryTextTheme.button),
+                ),
+                TextButton(
+                  onPressed: () {
+                    Utils.launchURL(Urls.clubEmail, AppIntl.of(context));
+                  },
+                  child: Text(AppIntl.of(context).email.toUpperCase(),
+                      style: Theme.of(context).primaryTextTheme.button),
+                ),
+              ]),
+            ]),
+      );
+
+  Widget _buildGradesCards(DashboardViewModel model, PreferencesFlag flag) =>
+      DismissibleCard(
+        key: UniqueKey(),
+        onDismissed: (DismissDirection direction) {
+          dismissCard(model, flag);
+        },
+        isBusy: model.busy(model.courses),
+        child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              Text(AppIntl.of(context).grades_title,
```

Comment:
You forgot to put the color. In light theme the title is white
![image](https://user-images.githubusercontent.com/37625944/117555864-e6153380-b030-11eb-9b1d-b7634f88a394.png)

Reply:
Oops ok je vais changer ca!